client/src/app/users/user.component.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { ComponentFixture, TestBed, async } from "@angular/core/testing";
+import { User } from "./user";
+import { UserComponent } from "./user.component";
+import { UserListService } from "./user-list.service";
+import { Observable } from "rxjs";
+import { PipeModule } from "../../pipe.module";
+
+describe("User component", () => {
+
+    let userComponent: UserComponent;
+    let fixture: ComponentFixture<UserComponent>;
+
+    let userListServiceStub: {
```

Comment:
Why does this need to be here, but the getUsers is not here? I would have thought either both would be needed or neither would be needed?